man/smartdc/man/man1/sdc-factoryreset.1.md

```diff
@@ -12,14 +12,15 @@ This command resets a machine to its originally installed state.  Specifically,
 it reboots the machine, imports all ZFS pools, and destroys them individually.
 It does this by setting a ZFS user property on the system pool. Using the
 `-s` flag will instead shut down the machine; putting it into a state where
-upon next reboot will import all ZFS pool and destroy them individually.
+upon next reboot, the machine will import all ZFS pool and destroy them
+individually.
 
 If this command is invoked unintentionally, an administrator can prevent the
 system from resetting itself by booting in rescue mode (noimport=true as a
 loader boot option) and clearing the smartdc:factoryreset property from the
 var dataset.  Use of the `-s` option makes this easier to accomplish.
 
-If the affected system boots without the noimport=true GRUB option, the only
+If the affected system boots without the noimport=true loader option, the only
```

Comment:
Maybe just say boot option? That way it’s boot loader agnostic.

Reply:
Will fix.